content-security-policy/support/testharness-helper.sub.js

```diff
@@ -0,0 +1,47 @@
+const Host = {
+  SAME_ORIGIN: "same-origin",
+  CROSS_ORIGIN: "cross-origin",
+};
+
+const PolicyHeader = {
+  CSP: "echo-policy.py?policy=",
+  EMBEDDING_CSP: "echo-embedding-csp.py",
+} 
+
+function generateURL(host, path) {
+  var url = new URL("http://{{host}}:{{ports[http][0]}}/content-security-policy/support/");
```

Comment:
I tried changing everything to `https`, but then I get "Aw, Snap!" page.
When I downgrade to `http` the last `for_each` test (cross origin redirect for cross origin iframe) is still failing :(